test/integration/manifests/cns/daemonset.yaml

```diff
@@ -41,7 +41,7 @@ spec:
           effect: NoSchedule
       containers:
         - name: cns-container
-          image: acnpublic.azurecr.io/azure-cns:v1.4.32
+          image: acnpublic.azurecr.io/azure-cns:v1.4.33
```

Comment:
lets use latest v1.4.35

Reply:
this is just a placeholder, the tests update the image to the latest working version on the branch